views/templates/admin/tabs/content/config_elements/cms.tpl

```diff
@@ -23,7 +23,7 @@
 *  International Registered Trademark & Property of PrestaShop SA
 *}
 
-<div class="form-group psr-cms {if $aBlock['type_link'] != $LINK_TYPE_CMS} inactive{/if}">
+<div class="form-group psr-cms {if (isset($aBlock) && $aBlock['type_link'] != $LINK_TYPE_CMS) || !isset($aBlock)} inactive{/if}">
```

Comment:
If we can, but it's only a bonus (😄) can we rename these variables ? They were created following [hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation) but I think it's not standard in php, so we can go back to a standard naming strategy instead

Reply:
So it would mean `$aBlock` => `$block` for example

Reply:
:ok_hand: